apps/publisher/modules/page-decorators.js

```diff
@@ -233,6 +233,14 @@ var pageDecorators = {};
             }
         }
     };
+    pageDecorators.taxonomyAvailability = function(ctx, page, utils) {
+        var app = require('rxt').app;
```

Comment:
Thank you Sameera for suggestion. But app.isTaxonomyEnabled() code block has been refactored with multiple taxonomy. Its not available anymore.